CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# MAXSwerve Java Template Changelog
+
+## [Unreleased]
+
+### Added
+
+- Added a configurable rate limiting system to prevent excessive loads from causing premature wheel failure.
+
+## [1.0] - 2023-01-18
```

Comment:
This is good, but the README should also clearly state which version of the template is in use.